web/app/routes/authenticated.ts

```diff
@@ -26,9 +30,10 @@ export default class AuthenticatedRoute extends Route {
     await this.authenticatedUser.loadInfo.perform();
 
     /**
-     * If the session is authenticated with the front- and back-ends,
-     * kick off the task to poll for expired auth.
+     * If using Google auth, kick off the task to poll for expired auth.
      */
-    void this.session.pollForExpiredAuth.perform();
+    if (!this.configSvc.config.skip_google_auth) {
+      void this.session.pollForExpiredAuth.perform();
```

Comment:
Why does skipping Google auth negate the need to poll for expiredAuth?

Reply:
Good q, 2 things here:
- I still need to do some more testing on the app behavior when Okta auth expires; it might still be useful to do this polling if the user isn't immediately redirected to log in to Okta.
- Currently `pollForExpiredAuth` is (understandably) Google-centric so I think we'll want to at least skip this until that either becomes neutral to the auth provider or has an Okta option built in too (assuming that we'll want to poll when skipping Google auth; still need to test).

Reply:
> it might still be useful to do this polling if the user isn't immediately redirected to log in to Okta

Yeah, I'd like to maintain our existing behavior where we detect passive logouts: If your token expires in the background, we should block [Okta]'s immediate-redirect function to show a gentler "reauathenticate" message.

Your change makes sense and its good to know you're planning to test further.

Reply:
Re-enabled polling auth for Okta in this commit: https://github.com/hashicorp-forge/hermes/pull/144/commits/23635e630b6d1407d67b129b907fa7d8dc1273b6